### DISCUSSION TITLE
PanzaUI Discussions

### DISCUSSION THREAD

**Discussion by MarioCatuogno** [MarioCatuogno/PanzaUI#58] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to share feedback and ideas. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by maikelschippers** [comment]
Nice setup, setting it up now. Guess your missing the minimap addon?

**Reply by MarioCatuogno** [reply]
Hi @maikelschippers! 
The minimap is from LeatrixPlus addon, there's an option to make it squared. Let me know if you try it and feel free to open issue for suggestions/bugs.

Thanks!
